src/pet.js

```diff
@@ -0,0 +1,73 @@
+const max_fitness = 10;
+const empty_belly = 0;
+const grow_old = 1;
+
+function Pet (name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = empty_belly;
+    this.fitness = max_fitness;
+    this.child = [];
+    };
+
+    Pet.prototype = {
+        get isAlive(){
+            return this.age < 30 && this.hunger < 10 && this.fitness > 0
+        }
+    };
+    
+    Pet.prototype.growUp = function (){
```

Comment:
This is missing some important functionality. See the track here: https://platform.manchestercodes.com/module/programming-fundamentals/oop/?pageId=Zpz2cKjJYn2oH2SmFZwz

Rather than having a separate function for getHungry as you have here, it should be included as part of this function, as well as decreasing the fitness rather than separating this out as you have now